root/static/scripts/common/i18n.js

```diff
@@ -137,46 +142,7 @@ exports.expandToArray = function (string, args) {
 
         return accum;
     }, []);
-};
-
-exports.commaList = function (items) {
-    var count = items.length;
-
-    if (count <= 1) {
-        return items[0] || "";
-    }
-
-    var output = l("{almost_last_list_item} and {last_list_item}", {
-        last_list_item: items[count - 1],
-        almost_last_list_item: items[count - 2]
-    });
-
-    items = items.slice(0, -2).reverse();
-    count -= 2;
-
-    for (var i = 0; i < count; i++) {
-        output = l("{list_item}, {rest}", { list_item: items[i], rest: output });
-    }
-
-    return output;
-};
-
-exports.commaOnlyList = function (items) {
-    var output = '';
-
-    if (!items.length) {
-        return output;
-    }
-
-    output = l('{last_list_item}', {last_list_item: items.pop()});
-    items.reverse();
-
-    _.each(items.reverse(), function (item) {
```

Comment:
Incorrectly reversed the list twice, which is why that one relationship editor test changed. I added some tests for commaOnlyList.